README.md

```diff
@@ -8,11 +8,11 @@ Benchmarks for Rust crypto libraries
 
 |                                              |       *ring*       |       Octavo       |     rust-crypto    | rust-nettle (Nettle) | rust-openssl (OpenSSL) | sodiumoxide (libsodium) | Windows CNG | Mac/iOS Common Crypto |
 |----------------------------------------------|:------------------:|:------------------:|:------------------:|----------------------|:----------------------:|:-----------------------:|:-----------:|:---------------------:|
-| SHA&#x2011;1 & SHA&#x2011;2                  | :white_check_mark: | :white_check_mark: | :white_check_mark: |                      | :white_check_mark:     |                         |             |                       |
+| SHA&#x2011;1 & SHA&#x2011;2                  | :white_check_mark: | :white_check_mark: | :white_check_mark: |                      | :white_check_mark:     | SHA-256/512 only        |             |                       |
```

Comment:
Please write this as "SHA-{256, 512}" only. The notation like SHA-512/256 refers to something else (truncated hashes).